kie-wb-smoke-tests/pom.xml

```diff
@@ -385,6 +391,27 @@
                 <groups>org.kie.smoke.wb.category.KieWbSeleniumSmoke</groups>
               </configuration>
             </plugin>
+            <plugin> <!-- Download & unzip IEDriverServer.exe - a binary required to run InternetExplorerDriver -->
+              <artifactId>maven-antrun-plugin</artifactId>
+              <executions>
+                <execution>
+                  <phase>pre-integration-test</phase>
+                  <goals>
+                    <goal>run</goal>
+                  </goals>
+                  <configuration>
+                    <target>
+                      <get src="http://selenium-release.storage.googleapis.com/2.48/IEDriverServer_x64_2.48.0.zip"
```

Comment:
+1. We need to figure out how to make this work as simple property replacement would work here (2.48 vs 2.48.0).